openpgp/packet/private_key.go

```diff
@@ -366,6 +374,24 @@ func (pk *PrivateKey) parseElGamalPrivateKey(data []byte) (err error) {
 	return nil
 }
 
+func (pk *PrivateKey) parseECDHPrivateKey(data []byte) (err error) {
```

Comment:
Yeah, it at least will allow reading of the key without crashing (what we used to do with an actual `panic`). I ought to go ahead and implement ECDH but it's tricky and takes a lot of tinkering, so I think it's worth it to put off for now, and at least get import/export of these keys working.